### PR TITLE
Blockchain Explorer

### PR DESCRIPTION
**Pull request by ArchanaArige**
Blockchain Explorer is a simple, powerful, easy-to-use, well-maintained, open-source utility to browse activity 
on the underlying blockchain network.

### CODE REVIEW

File: labs/blockchain-explorer.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+on the underlying blockchain network. 
+
+# Scope of Lab
+- Enhance the existing application in-terms of new features
+- Make it compatible with latest version of Hyperledger Fabric 
+- Uplift the user-interface with the robust design
+- Test-cases and documentation
+
+# Initial Committers
+- https://github.com/ryjones
```
Comment (arsulegai) [inline labs/blockchain-explorer.md:20]: @ArchanaArige please add the current set of maintainers that we want to start the labs with. These are previous committers, their contributions are counted but they would not be a maintainer as of now.

Comment (ArchanaArige) [reply labs/blockchain-explorer.md:20]: Hi @arsulegai. Thanks for the review. As per the proposal-template.md there are only five sections where we don't have #current maintainers. Please confirm whether we can add a separate section as #current maintainers to the proposal as we suggested.

Comment (arsulegai) [reply labs/blockchain-explorer.md:20]: @ArchanaArige as per the discussion outcome in discord. @tkuhrt too agrees that we can replace this list by adding current set of maintainers.

Comment (ArchanaArige) [reply labs/blockchain-explorer.md:28]: @arsulegai , I have added the current-maintainers section, in-addition to the initial-committers.
Although Hyperledger-Explorer coined ad EOL, it's good to have the initial-committers section who have contributed to this project initially. If there has to be any changes, please do let me know.

Comment (ArchanaArige) [reply labs/blockchain-explorer.md:28]: Adding AdityaJoshi to the current maintainers list.

**Review by tkuhrt — APPROVED** [review]

**Comment by nidhi-singh02** [comment]
Thank you for submitting this. Great initiative to revive the Explorer project. I would like to co-sponsor the lab!

**Comment by arigelas** [comment]
> Thank you for submitting this. Great initiative to revive the Explorer project. I would like to co-sponsor the lab!

Thanks Nidhi for your interest as co-sponsor. We will update the proposal file accordingly

**Comment by ArchanaArige** [comment]
> > Thank you for submitting this. Great initiative to revive the Explorer project. I would like to co-sponsor the lab!
> 
> Thanks Nidhi for your interest as co-sponsor. We will update the proposal file accordingly. @ArchanaArige will be updating the file as required.

@arigelas , Updated the file as suggested. Thanks.

**Review by nidhi-singh02 — APPROVED** [review]

**Pull request merged**